deRSE23/README.md

```diff
@@ -0,0 +1,19 @@
+# deRSE23 Submission
+
+Abstract (and slides) for the [deRSE23](https://de-rse23.sciencesconf.org/) conference.
+
+* Title: Fortran-Lang: an open source Fortran community
+* Presenter: Ivan Pribec
+* Co-authors: Vincent Magnin, Asdrubal Lozada-Blanco, Ondrej Certik, Jeremie Vandenplas
+
+## Abstract
+
+A thriving community is vital for the success and continuity of a programming language. 
+Despite the importance of Fortran in science and engineering, and evolution of the language itself,
+attracting new users and retaining existing ones has proven challenging for Fortran.
+
+In this talk, I will present the efforts of the Fortran-lang community including the Fortran package manager, standard library, website, LFortran compiler and other tools. 
+Fortran-lang aims to provide a central place for Fortran users,
+complementing the language standardization committee and compiler vendor communities.
+Building upon the common best practices in software development, 
+we aim to reduce the language learning curve and promote Fortran adoption.
```

Comment:
Thank you for leading this effort. I think the abstract is in very good shape already. The only significant issue I found was that there was no single, to-the-point statement of a solution that immediately followed the problem statement (2nd sentence). I added the 3rd sentence to address this. Of course, to stay under 750 chars, I had to get creative in the remainder of the abstract and reword things a bit, without a change in content. I also removed the paragraph break--for an abstract this short, I think it reads best as a single paragraph. I hope you like the suggested changes, but I'm overall okay with the previous version as well.

```suggestion
A thriving community is vital for the success and continuity of a programming language. Despite Fortran’s role in science and engineering, and evolution of the language itself, attracting new users and retaining existing ones has been a challenge for Fortran. To address this, an open-source community Fortran-lang has formed in 2019. Here, we present the efforts of Fortran-lang such as the Fortran package manager, standard library, website, LFortran compiler and other tools. Fortran-lang aims to provide a central place for Fortran users, complementing the language standardization and compiler communities. Building on the best practices in software development, we aim to reduce the language learning curve and promote Fortran adoption.
```

Reply:
Thanks Milan! I agree with the problem-solution tension. In addition to the paragraph split you've shaved off a few characters by getting rid of the linebreaks 😄 .